packages/ndla-ui/src/locale/messages-en.ts

```diff
@@ -1326,6 +1326,50 @@ const messages = {
       external: 'External resource',
     },
   },
+  gloss: {
+    examples: 'Examples',
+    Transcriptions: {
+      traditional: 'Traditional',
+      pinyin: 'Pinyin',
+    },
```

Comment:
Treng nok ikkje dette ekstra nivået.
```suggestion
    traditional: 'Traditional',
    pinyin: 'Pinyin',
```

Reply:
Dvs du kan alltids la det være der, men bruk heller små bokstaver.